frontend/src/libs/components/card/card.tsx

```diff
@@ -20,37 +22,55 @@ const Card: React.FC<Properties> = ({
   onClick,
   isActive = false,
   iconName,
+  iconRight,
+  onIconClick,
   iconColor,
 }) => {
   const hasNoImageOrIcon = !imageUrl && !iconName;
   const hasImage = Boolean(imageUrl);
   const hasIcon = Boolean(iconName);
 
   return (
-    <button
-      className={getValidClassNames(
-        styles['item'],
-        isActive && styles['selected'],
-        hasNoImageOrIcon && styles['no-image'],
-      )}
-      onClick={onClick}
-    >
-      {!hasNoImageOrIcon && (
-        <div className={styles['image-container']}>
-          {hasImage && (
-            <div className={styles['image-placeholder']}>
-              <img src={imageUrl} alt="not found" className={styles['image']} />
-            </div>
-          )}
-          {hasIcon && (
-            <div className={styles['icon-background']}>
-              <Icon name={iconName as IconName} color={iconColor} />
+    <>
+      <button
+        className={getValidClassNames(
+          styles['item'],
+          isActive && styles['selected'],
+          hasNoImageOrIcon && styles['no-image'],
+        )}
+        onClick={onClick}
+      >
+        <div className={styles['item-info']}>
+          {!hasNoImageOrIcon && (
+            <div className={styles['image-container']}>
+              {hasImage && (
+                <div className={styles['image-placeholder']}>
+                  <img src={imageUrl} alt="Card" className={styles['image']} />
+                </div>
+              )}
+              {hasIcon && (
+                <div className={styles['icon-background']}>
+                  <Icon name={iconName as IconName} color={iconColor} />
+                </div>
+              )}
             </div>
           )}
+          <div className={styles['title']}>{title}</div>
         </div>
+      </button>
+      {iconRight && (
```

Comment:
Add boolean for it

Reply:
@v0ldemar01 But if I make it a boolean, I need to pass IconName to have the name of the icon, so in this way, the code under will be displayed, but I don't need this icon there, so as a result I see this big icons between picture and title, but I don't want it. So I think my previous solution looks ok. What do you think?
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/102380564/0b8478b8-a556-4bef-a3c9-3723df350ebb)
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/102380564/29ff583f-fb66-4456-965c-4df6ba3c9464)

Reply:
const hasIconRight = Boolean(iconRight) of something like that or actionIcon looks better

Reply:
Now I see your point, agree.